plugins/ci/skills/fetch-tide-pr-status/test_fetch_tide_pr_status.py

```
@@ -0,0 +1,391 @@
+        "number": 1,
+        "title": "Test PR",
+        "user": {"login": "dev"},
+        "html_url": f"https://github.com/{repo}/pull/1",
+        "state": "open",
+        "draft": draft,
+        "mergeable": mergeable,
+        "base": {
+            "ref": branch,
+            "repo": {"full_name": repo},
+        },
+        "labels": [{"name": l} for l in (labels or [])],
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)